packages/devtools_app/lib/src/shared/server/server.dart

```diff
@@ -25,15 +25,22 @@ part '_dtd_api.dart';
 part '_extensions_api.dart';
 part '_preferences_api.dart';
 part '_release_notes_api.dart';
+part '_server_api.dart';
 part '_survey_api.dart';
 
 final _log = Logger('devtools_server_client');
 
-/// Whether the DevTools server is available.
+/// Whether the DevTools server is available so that the HTTP API can be used.
+///
+/// A value of `true` here does not necessarily mean the legacy SSE API is
+/// available.
 ///
 /// Since the DevTools server is a web server, it is only available for the
 /// web platform.
 ///
+/// TODO(dantup): Since this relates only to non-SSE API, it could be available
+///  for non-web?
```

Comment:
I'm not sure of the logic behind this comment, but wonder if maybe it was because of SSE (and maybe `package:sse` only worked for web?). I feel like we should (in theory) be able to run the server and connect a desktop version of DevTools to it, which makes this comment incorrect?